core/src/consensus/tendermint/stake/action_data.rs

```diff
@@ -33,11 +33,11 @@ lazy_static! {
         ActionDataKeyBuilder::new(CUSTOM_ACTION_HANDLER_ID, 1).append(&"StakeholderAddresses").into_key();
 }
 
-pub type StakeBalance = u64;
+pub type StakeQuantity = u64;
```

Comment:
While it is not about this PR, what is the maximum amount of stake?
I think the JavaScript sdk can safely use `number` if this type is u32.